cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/worker/system/SystemWorkerTwillRunnable.java

```diff
@@ -156,7 +158,8 @@ protected void bindKeyManager(Binder binder) {
         new AuthorizationModule(),
         new AuthorizationEnforcementModule().getMasterModule(),
         new AuditLogWriterModule(cConf).getDistributedModules(),
-        Modules.override(new AppFabricServiceRuntimeModule(cConf).getDistributedModules())
+        Modules.override(new AppFabricServiceRuntimeModule(cConf, EnumSet.allOf(ServiceType.class))
```

Comment:
Could you create an constant for `EnumSet.allOf(ServiceType.class)` in the AppFabricServiceRuntimeModule? This should make this lines less cluttered. Or, alternatively, move to factory methods (e.g. `AppFabricServiceRuntimeModule.for(ServiceType.PROCESSOR)` and `AppFabricServiceRuntimeModule.forCombinedService()` or similar?

Reply:
Done.